src/components/AppHeaderCenter.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+export const AppHeaderCenter = styled.div`
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 50%;
+    @media only screen and (max-width: 415px) {
```

Comment:
415px and const variable in some styled-components variables, also you can add it to theme object and read there from props.

Reply:
415px width use as variable now